src/Sulu/Bundle/PageBundle/Content/Types/DateTime.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PageBundle\Content\Types;
+
+use PHPCR\NodeInterface;
+use Sulu\Component\Content\Compat\PropertyInterface;
+use Sulu\Component\Content\SimpleContentType;
+
+/**
+ * ContentType for DateTime.
+ */
+class DateTime extends SimpleContentType
+{
+    const FORMAT = 'Y-m-d\TH:i:s';
+
+    public function __construct()
+    {
+        parent::__construct('DateTime');
+    }
+
+    public function write(
+        NodeInterface $node,
+        PropertyInterface $property,
+        $userId,
+        $webspaceKey,
+        $languageCode,
+        $segmentKey
+    ) {
+        $value = $property->getValue();
+        if (null != $value) {
+            $value = \DateTime::createFromFormat(static::FORMAT, $value);
+
+            $node->setProperty($property->getName(), $value);
+        } else {
+            $this->remove($node, $property, $webspaceKey, $languageCode, $segmentKey);
+        }
+    }
+
+    public function read(NodeInterface $node, PropertyInterface $property, $webspaceKey, $languageCode, $segmentKey)
+    {
+        $value = '';
+        if ($node->hasProperty($property->getName())) {
+            /** @var \DateTime $propertyValue */
+            $propertyValue = $node->getPropertyValue($property->getName());
+
+            if ($propertyValue instanceof \DateTime) {
+                $value = $propertyValue->format(static::FORMAT);
```

Comment:
I think formatting the value like this might be a bit impractical for the website output... In the twig template you'll receive a string value like `"2020-08-21T23:05:00"`. This  makes it harder than it should be to align the output with the given language. I would suggest we override the `getContentData` method of the `SimpleContentType` class, and return a `DateTime` value there.

Reply:
Oh, and please don't forget to write a test for that as well :slightly_smiling_face:

Reply:
done 🙂